src/masonite/routes/HTTPRoute.py

```diff
@@ -30,7 +29,12 @@ def __init__(
         self.controller_method = None
         self._domain = None
         self._name = name
+
         self.request_method = [x.lower() for x in request_method]
+        # add HEAD method if GET
+        if "get" in self.request_method and "head" not in self.request_method:
```

Comment:
This part is confusing to me. Is there another way to do this? Seems weird automatically adding head to every GET route

Reply:
That 's how Laravel is handling it and that's quite mandatory because each GET route should be able to be accessed through GET !